src/activate/activate.c

```diff
@@ -384,16 +381,14 @@ static int parse_argv(int argc, char *argv[]) {
 
                 case 'd':
                         if (arg_socket_type == SOCK_SEQPACKET)
-                                return log_error_errno(SYNTHETIC_ERRNO(EINVAL),
-                                                       "--datagram may not be combined with --seqpacket.");
+                                return log_error_errno(SYNTHETIC_ERRNO(EINVAL), "--datagram may not be combined with --seqpacket.");
```

Comment:
The line seems to be long a bit. We use 109 characters per line. See c90ee83400783739ac6df9f12c43bd7074dea8fa.

Reply:
ok, i can change this in the settings...